vmdb/app/controllers/application_controller/current_user.rb

```diff
@@ -0,0 +1,77 @@
+module ApplicationController::CurrentUser
+  extend ActiveSupport::Concern
+
+  included do
+    helper_method :current_user,  :current_userid, :current_username
+    helper_method :current_group, :current_groupid, :current_group_description
+  end
+
+  def clear_current_user
+    self.current_user = nil
+  end
+  protected :clear_current_user
```

Comment:
minor, but no need to mark every method protected individually.  You can put the word protected on like 8 and you're good.  Not really important though.